osu.Game/Beatmaps/Formats/LegacyStoryboardDecoder.cs

```diff
@@ -65,12 +64,14 @@ protected override void ParseLine(Storyboard storyboard, Section section, string
         private void handleEvents(string line)
         {
             var depth = 0;
+            var lineSpan = line.AsSpan();
 
-            while (line.StartsWith(" ", StringComparison.Ordinal) || line.StartsWith("_", StringComparison.Ordinal))
+            while (lineSpan.StartsWith(" ", StringComparison.Ordinal) || lineSpan.StartsWith("_", StringComparison.Ordinal))
             {
+                lineSpan = lineSpan.Slice(1);
                 ++depth;
-                line = line.Substring(1);
             }
+            line = line.Substring(depth);
```

Comment:
Maybe just `lineSpan.ToString()`.

Reply:
I tried both, and using `Substring` seems to be a bit faster (difference is within margin of error though). Should I still change to `lineSpan.ToString()`?

Reply:
If `Substring` doesn't have a significant performance advantage over even thousands of executions I'd change for the sake of readability.